src/Microsoft.AspNet.Mvc.Razor/LanguageViewLocationExpander.cs

```diff
@@ -62,7 +59,6 @@ public void PopulateValues(ViewLocationExpanderContext context)
             foreach (var location in viewLocations)
             {
                 yield return location.Replace("{0}", value + "/{0}");
-                yield return location;
```

Comment:
I am adding it in [Line 49](https://github.com/aspnet/Mvc/pull/2645/files#diff-13c0456c912173b92459976512739c7eR49). I had to remove it from here since it is getting added multiple times.